File: app/src/main/java/com/cartenz/cartenztaxcore/api/dao/LoginDao.java
package com.cartenz.cartenztaxcore.api.dao;

import com.cartenz.core.api.BaseApiDao;
import com.google.gson.JsonArray;


public class LoginDao extends BaseApiDao<LoginDao.DataDao> {

    public static class DataDao {
        public int systemId;
        public int userId;
        public String userName;
        public String fullName;
        public String email;
        public JsonArray roles;
        public JsonArray profiles;
        public JsonArray authorizations;
        public boolean superAdministrator;
        public boolean administrator;
        public String uuid;
        public boolean lockedOut;
        public boolean deleted;
        public boolean active;
        public String token;
    }
}

File: kotlinapp/src/main/java/com/cartenz/kotlinapp/api/dao/LoginDao.kt
package com.cartenz.kotlinapp.api.dao

import com.cartenz.kotlin_core.api.BaseApiDao
import com.google.gson.JsonArray

class LoginDao : BaseApiDao<LoginDao.DataDao>() {

    class DataDao {
        var systemId: Int = 0
        var userId: Int = 0
        var userName: String? = null
        var fullName: String? = null
        var email: String? = null
        var roles: JsonArray? = null
        var profiles: JsonArray? = null
        var authorizations: JsonArray? = null
        var superAdministrator: Boolean = false
        var administrator: Boolean = false
        var uuid: String? = null
        var lockedOut: Boolean = false
        var deleted: Boolean = false
        var active: Boolean = false
        var token: String? = null
    }
}
